packages/access-cice/package.py

```diff
@@ -36,6 +36,12 @@ class AccessCice(CMakePackage):
         description="CICE IO Method"
     )
 
+    variant("driver",
+            default="none",
+            values=("nuopc/cmeps", "access/cmeps", "standalone/cice"),
+            description="CICE driver path"
+            )
+
```

Comment:
I think either set a default or add something like this:

```suggestion
conflicts(
        "driver=none",
        msg=f"A configurations variant must be set, can be one of ..."
    )
```

Reply:
I think default sounds good as to not interfere with OM3!